config/initializers/devise.rb

```diff
@@ -1,3 +1,18 @@
+require 'devise_remote_user'
+
+DeviseRemoteUser.configure do |config|
+  config.env_key = lambda do |env|
+    if env['REMOTE_USER']
+      "#{env['REMOTE_USER']}@stanford.edu"
+    elsif Rails.env.development? && ENV['REMOTE_USER']
+      ENV['REMOTE_USER']
+    end
+  end
+  config.auto_create = true
+  config.auto_update = true
+  config.attribute_map = { webauth_groups: 'WEBAUTH_LDAPPRIVGROUP' }
```

Comment:
Ah, you know what, I think this added for Triannon work (where we were on a blessed server).

I can certainly take it out, but it's just no-op if we're not blessed, so maybe we could :shipit:.